src/taxon.jl

```diff
@@ -80,3 +80,92 @@ function taxon(name::String;
    end
 
 end
+
+"""
+**Get information about a taxon at any level using taxonID**
+
+    taxon(id::Int)
+
+This function will look for a taxon by its taxonID in the GBIF
+reference taxonomy.
+
+Optional arguments are
```

Comment:
You can remove pretty much everything, since this function will not accept keyword arguments.

Reply:
Ok, will do, I though adding that stuff still made sense